infra/terraform/hash/bastion.md

```diff
@@ -0,0 +1,16 @@
+# Bastion hosts
+
+The Bastion hosts defined through this configuration is used for SSH-ing into the private subnet of our AWS VPC.
+Bastion hosts are spun up per environment (e.g. `prod` and `dev`) and connect using current terraform outputs.
+
+It is possible to SSH into the Bastion host by running the `./ssh_bastion.sh` script.
+This script will use the currently selected terraform workspace to SSH into the appropriate host.
+
+The Bastion host can also act as a jump host to SSH tunnel a service to your local machine.
+For example to forward the RDS instance in the private subnet to your local machine, run
```

Comment:
No strong feelings but I think we don't use newlines in other `*.md`s inside of a paragraph
```suggestion
The Bastion hosts defined through this configuration is used for SSH-ing into the private subnet of our AWS VPC. Bastion hosts are spun up per environment (e.g. `prod` and `dev`) and connect using current terraform outputs.

It is possible to SSH into the Bastion host by running the `./ssh_bastion.sh` script. This script will use the currently selected terraform workspace to SSH into the appropriate host.

The Bastion host can also act as a jump host to SSH tunnel a service to your local machine. For example to forward the RDS instance in the private subnet to your local machine, run
```